editoast/src/modelsv2/connection_pool.rs

```diff
@@ -0,0 +1,6 @@
+use diesel_async::AsyncPgConnection;
+use diesel_async::pooled_connection::deadpool::Pool;
+
+pub type Connection = AsyncPgConnection;
```

Comment:
`modelsv2::Connection` feels insufficient in terms of naming. We're in the `modelsv2` module, so we probably expect objects of the model: so my first thought would be what does it connect? 2 railways? 2 parts of a train? But certainly not thinking of database. So I would rather have a longer name making this explicit: `DatabaseConnection` (*), `DbConnection`, `PostgresConnection`, `PgConnection`?

(*) I prefer `DatabaseConnection` since it's not an acronym (unlike `DbConnection`) and it's not tied to a particular technology (not mentioning Postgres or SQL server, or whatever DB system).

Another possibility would be to not expose it from the parent module. For now, you're reexposing it from `modelsv2` module so to import it, it's `crate::modelsv2::Connection` which gives the impression it's part of the objects in the model. If you do not reexpose, then importing it would be `crates::modelsv2::connection_pool::Connection` which might help making it more explicit it's something else.